src/app/chat/component/new-message-window/new-message-window.component.ts

```diff
@@ -46,15 +47,19 @@ export class NewMessageWindowComponent implements OnInit, AfterViewInit, OnDestr
   uploadedFile: File;
   @Input() class: string;
   @ViewChild('chat') chat: ElementRef;
+  currentPath: string;
 
   constructor(
     public chatsService: ChatsService,
     private commonService: CommonService,
     private socketService: SocketService,
     public userService: UserService,
     private jwt: JwtService,
-    public dialog: MatDialog
-  ) {}
+    public dialog: MatDialog,
+    private router: Router
+  ) {
+    this.currentPath = this.router.url;
```

Comment:
Please don't write any code in constructor of component because it may lead to errors. Use ngOnInit instead.